recipes/pygwb/meta.yaml

```
@@ -0,0 +1,54 @@
+{% set name = "pygwb" %}
+{% set version = "1.5.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/pygwb-{{ version }}.tar.gz
+  sha256: 675da6be45411e39b0032913499de3189abcf2f3bc4e48ff3bd208f86d22fa1d
+
+build:
+  skip: true  # [py<38]
```

This can be removed, conda-forge doesn't built py<38.

```suggestion
```

---

You probably want to infact replace this with

```suggestion
  skip: true  # [win or py>=312]
```

since the dependencies don't resolve on python 3.12, or on Windows.

---

done! thanks